### DISCUSSION TITLE
Wait until full per station?

### DISCUSSION THREAD

**Discussion by Voidheartd** [Erabior/RouteManager#92] [Ideas]
It would be nice to be able to control wait until full per station. For example, it's only useful for my passenger train to wait at Bryson and Whittier right now, not at Ela, Hemingway, and Alarka Jct.
There could be a wait until full checkbox next to each station on the list, similar to the location of pause for refuel next to enable route mode.

**Comment by AdamC1228** [comment]
This will be considered once we resolve the long term plans for how best to handle the UI situation.
